web/src/containers/activity/CostAllocateFFPQuarterly.js

```diff
@@ -121,7 +116,7 @@ class CostAllocateFFPQuarterly extends Component {
                             headers={`act_qbudget_fy${year} act_qbudget_fy${year}_q${q}`}
                           >
                             <Dollars>
-                              {quarterlyFFP[year].subtotal[name].dollars}
+                              {quarterlyFFP[year][q][name].dollars}
```

Comment:
A few small display bugs in here.  The cost allocation quarterly table was showing the computed subtotal everywhere, rather than the individual quarterly amounts.

Reply:
good catch!